android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
are you trying to get `notification` property when user click on the banner? why?